### ISSUE TITLE
Adjust size of the windows?

### ISSUE THREAD

**Issue by ankidoc** [rad4day/anki-editor-preview-plugin#3]
Hello! just wanted to check out, if there is possibility in the feature that one can change the default size of the previewer window since it's taking a huge space and 50/50 would be better. So with every opening I have to adjust the size.

Would be happy when you reply, thanks!

Have a nice week

**Comment by nickdvlpr** [comment]
@ankidoc 
If you are okay with completely hiding the original editor windows with your fields, you can add this line:
**`editor.web.setVisible(False)`**

after 
`layout.insertWidget(web_index, split)`

**Comment by nickdvlpr** [comment]
@ankidoc 
A simple way I found to keep both views but swap it so that the Live Preview is on top and always showing is to reorder two lines of code and add one with a minimum height:
```
        split.addWidget(editor.webview) # this is the Live Preview half of the split
        editor.webview.setMinimumHeight(500)
        split.addWidget(editor.web) # this is the editor fields (default Anki look) half of the split
```

**Comment by rad4day** [comment]
This should have be implemented with PR #7 just now, if it wasn't before.
